services/movie_session.py

```diff
@@ -42,3 +42,10 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
+
+
+def get_taken_seats(
+        movie_session_id: int
+) -> list[dict]:
+    return list(Ticket.objects.filter(
+        movie_session_id=movie_session_id).values("row", "seat"))
```

Comment:
In case you do not have any arguments on the same line as the opening parenthesis, your closing parenthesis should not have any arguments on the same line as well. Change your code style to encounter that.